attributes/default.rb

```diff
@@ -43,6 +43,14 @@
 }
 default['consul']['source_revision'] = 'master'
 
+# GOMAXPROCS should be greater than 1, see
+# https://groups.google.com/forum/#!topic/consul-tool/qewFEqgAoF8
+if node['cpu']['total'] > 1
+  default['consul']['gomaxprocs'] = node['cpu']['total']
```

Comment:
So the problem that we're trying to solve here is if there is only a single CPU in an instance that we set this greater than one?